spot-contracts/contracts/RolloverVault.sol

```diff
@@ -125,51 +108,98 @@ contract RolloverVault is
     /// @notice The perpetual token on which rollovers are performed.
     IPerpetualTranche public perp;
 
+    //--------------------------------------------------------------------------
+    // v2.0.0 STORAGE ADDITION
+
+    /// @notice External contract that controls fees for minting, redeeming, swapping and deployment.
```

Comment:
Maybe just say that this orchestrates fees across the spot protocol. If we list specific fees, those can get out of date.